projects/cloud_feedbacks/index.md

```diff
@@ -1,463 +1,834 @@
----
-layout: cloud_feedbacks
-title: Cloud Feedbacks
----
+--- layout: cloud_feedbacks title: Cloud Feedbacks ---
```

Comment:
@mzelinka The Jekyll front matter at the top of the page needs to have the triple dashed lines at the top and bottom of the section, and have its variables on separate lines.  Otherwise, Jekyll will not process it correctly.
```suggestion
--- 
layout: cloud_feedbacks 
title: Cloud Feedbacks 
---
```